.gitignore

```diff
@@ -4,20 +4,8 @@ dist
 # Coverage Report
 coverage
 
-# docma docs
-docs/website
-
-# Logs
-logs
-*.log
-*.log.*
-
 # Dependency directory
 node_modules
 
-# SSL certificates
-.ssl
-
```

Comment:
This was most certainly an artifact of when we had the website hosted from this repo, since that is no longer a thing, we can remove this